docs/settings/telemetry-settings.asciidoc

```diff
@@ -4,32 +4,30 @@
 <titleabbrev>Telemetry settings</titleabbrev>
 ++++
 
-By default, Usage Collection (also known as Telemetry) is enabled. This
-helps us learn about the {kib} features that our users are most interested in, so we
-can focus our efforts on making them even better.
+Usage Collection (also known as Telemetry) is enabled by default. This allows us to learn what our users are most interested in, so we can improve our products and services.
+
+Refer to our https://www.elastic.co/legal/product-privacy-statement[Privacy Statement] to learn more.
 
 You can control whether this data is sent from the {kib} servers, or if it should be sent
 from the user's browser, in case a firewall is blocking the connections from the server. Additionally, you can decide to completely disable this feature either in the config file or in {kib} via *Management > Kibana > Advanced Settings > Usage Data*.
 
-Refer to our https://www.elastic.co/legal/product-privacy-statement[Privacy Statement] to learn more.
-
 [float]
 [[telemetry-general-settings]]
 ==== General telemetry settings
 
-`telemetry.sendUsageFrom`::
-  Set to `'server'` to report the cluster statistics from the {kib} server.
-  If the server fails to connect to our endpoint at https://telemetry.elastic.co/, it assumes
-  it is behind a firewall and falls back to `'browser'` to send it from users' browsers
-  when they are navigating through {kib}. Defaults to `'server'`.
```

Comment:
I moved it to the bottom because it feels like it's the least likely setting to be used.

Happy to move it back to the top if you think it's worth highlighting it first.